uvvid/cli.py

```diff
@@ -30,13 +31,22 @@ def view(ctx, cursor, video):
         if ret is False:
             break
 
-        coord = uvvid.find_cursor(frame, template_frame, debug=debug)
+        coord = uvvid.find_cursor(frame, template_frame)
 
         if prev_frame is not None:
-            uvvid.is_cursor_drawing(frame,
-                                    prev_frame,
-                                    coord,
-                                    debug=debug)
+            drawing, color = uvvid.is_cursor_drawing(frame, prev_frame, coord)
+            if debug:
+                top_left = (coord[0] - template_frame.shape[0]//2,
```

Comment:
@Foxiy I am OK with this staying in here but if it gets any bigger, I believe we should move it to a separate function. Or maybe two (`draw_cursor` and `draw_cursor_color_state`).